### PR TITLE
documentation(DOC-4094): Reviewed

### PR DESCRIPTION
**Pull request by laksmipriyamurugesan**
DOC-4094-WinUI-UG-Content-Set2

### CODE REVIEW

**Comment by laksmipriyamurugesan** [comment]
rebuild

**Comment by laksmipriyamurugesan** [comment]
rebuild

**Comment by laksmipriyamurugesan** [comment]
rebuild

**Comment by laksmipriyamurugesan** [comment]
rebuild

**Comment by laksmipriyamurugesan** [comment]
rebuild

File: winui/Badge/Alignment-positioning.md [hunk 1]
```diff
@@ -211,9 +211,9 @@ badge.VerticalPosition = 0.8;
...
+By using the `HorizontalAlignment`, `VerticalAlignment`, `HorizontalAnchor`, and `VerticalAnchor` properties, you can place `Badge` only at particular position. If you want to place the `Badge` at a custom position, use the `HorizontalPosition` and `VerticalPosition` properties, as well as the [HorizontalAnchorPosition](https://help.syncfusion.com/cr/winui/Syncfusion.UI.Xaml.Notifications.SfBadge.html#Syncfusion_UI_Xaml_Notifications_SfBadge_HorizontalAnchorPosition) and [VerticalAnchorPosition](https://help.syncfusion.com/cr/winui/Syncfusion.UI.Xaml.Notifications.SfBadge.html#Syncfusion_UI_Xaml_Notifications_SfBadge_VerticalAnchorPosition) properties. This will effective only on by setting the `HorizontalAnchor` and `VerticalAnchor` properties value as `Custom`. The value range for the `HorizontalAnchorPosition` and `VerticalAnchorPosition` properties is `0` to`1`. The default value of `HorizontalAnchorPosition`and `VerticalAnchorPosition` properties is `0`.
```
Comment (mohamedks) [inline winui/Badge/Alignment-positioning.md:214]: This will be effective.

File: winui/Badge/Alignment-positioning.md [hunk 2]
```diff
@@ -211,9 +211,9 @@ badge.VerticalPosition = 0.8;
...
 
-For example, you will see how the  `HorizontalPosition` and `HorizontalAnchorPosition` values working based on the `VerticalAnchorPosition` and `VerticalPosition` values. Here,  `VerticalPosition` and `VerticalAnchorPosition` properties value is `0`.
+For example, you will see how the  `HorizontalPosition` and `HorizontalAnchorPosition` values works based on the `VerticalAnchorPosition` and `VerticalPosition` values. Here,  `VerticalPosition` and `VerticalAnchorPosition` properties value is `0`.
```
Comment (mohamedks) [inline winui/Badge/Alignment-positioning.md:216]: values work

File: winui/Badge/Alignment-positioning.md [hunk 3]
```diff
@@ -326,11 +326,11 @@ badge.Content = "99+";
 
 ![Alignment of the Badge content is changed](Getting-Started_images/Content_Alignment.png)
 
-N> Download demo application from [GitHub](https://github.com/SyncfusionExamples/syncfusion-winui-badge-control-examples/blob/master/Samples/Badge_Features)
+N> Download demo application from [GitHub](https://github.com/SyncfusionExamples/syncfusion-winui-badge-control-examples/blob/master/Samples/Badge_Features).
 
 ## Padding for Badge content
 
-By default, `Badge` control size automatically assigned based on its content. You can change the left, right, top, bottom content spacing of a `Badge` control by using the `Padding` property. The default value of `Padding` property is `0,0,0,0`.
+By default, `Badge` control size automatically assigned based on their content. You can change the left, right, top, and bottom content spacing of a `Badge` control by using the `Padding` property. The default value of `Padding` property is `0,0,0,0`.
```
Comment (mohamedks) [inline winui/Badge/Alignment-positioning.md:333]: its content

File: winui/Barcode/Symbology-Types.md [hunk 4]
```diff
@@ -178,11 +178,11 @@ The [Code32Barcode](https://help.syncfusion.com/cr/winui/Syncfusion.UI.Xaml.Barc
...
 
 ### Code 39
-The [Code39Barcode](https://help.syncfusion.com/cr/winui/Syncfusion.UI.Xaml.Barcode.Code39Barcode.html) is a Barcode Symbology that encodes alphanumeric characters into a series of bars. It may be of any length, although more than 25 characters begin to push the bounds. This Symbology is the only type of the Barcode in common use that does not require a checksum.
+The [Code39Barcode](https://help.syncfusion.com/cr/winui/Syncfusion.UI.Xaml.Barcode.Code39Barcode.html) is a type of Barcode Symbology that encodes alphanumeric characters into a series of bars. It can be any length, though more than 25 characters begin to push the bounds. This is the only type of Barcode symbology that does not require a checksum.
 
 * Allows character set of digits (0-9), upper case alphabets (A-Z), and symbols like space, minus (-), plus (+), period (.), dollar sign ($), slash (/), and percent (%).
 * Always starts and ends with an asterisk (*) symbol, termed as start and stop character.
-* Each character is encoded with 5 bars and 4 spaces where 3 are wide and 6 are narrow.
+* Each character is encoded with 5 bars and 4 spaces, each of which is 3 are wide and 6 are narrow.
```
Comment (mohamedks) [inline winui/Barcode/Symbology-Types.md:185]: The sentence seems to be wrong now.

File: winui/Calendar-Date-Picker/Date-Selection-And-Restriction.md [hunk 5]
```diff
@@ -26,9 +26,9 @@ sfCalendarDatePicker.SelectedDate = new DateTimeOffset(new DateTime(2021, 01, 06
 
 ![programatic-date-selection-in-winui-calendar-date-picker](Getting-Started_images/programatic-date-selection-in-winui-calendar-date-picker.png)
 
-N> Download demo application from [GitHub](https://github.com/SyncfusionExamples/syncfusion-winui-tools-calendardatepicker-examples/blob/main/Samples/Restriction)
+N> Download demo application from [GitHub](https://github.com/SyncfusionExamples/syncfusion-winui-tools-calendardatepicker-examples/blob/main/Samples/Restriction).
 
-You can also change the selected date interactively by selecting a date from dropdown calendar or by entering the date value in editor of `Calendar Date Picker`. You can get the selected date from the `SelectedDate` property.
+You can also change the selected date interactively by selecting a date from drop-down calendar or by entering the date value in editor of `Calendar Date Picker`. Now, you can get the selected date from the `SelectedDate` property.
```
Comment (mohamedks) [inline winui/Calendar-Date-Picker/Date-Selection-And-Restriction.md:31]: **Now** is not meaningful here.

File: winui/Calendar-Date-Picker/Date-Selection-And-Restriction.md [hunk 6]
```diff
@@ -184,7 +184,7 @@ private void SfCalendarDatePicker_CalendarItemPrepared(object sender, CalendarIt
 
 ## Highlight today and selected dates
 
-You can highlight the today and selected date in dropdown calendar using [SelectionHighlightMode](https://help.syncfusion.com/cr/winui/Syncfusion.UI.Xaml.Calendar.SfCalendarDatePicker.html#Syncfusion_UI_Xaml_Calendar_SfCalendarDatePicker_SelectionHighlightMode) property to update the background and border of the dates. The default value of `SelectionHighlightMode` property is **Outline**.
+You can use the [SelectionHighlightMode](https://help.syncfusion.com/cr/winui/Syncfusion.UI.Xaml.Calendar.SfCalendarDatePicker.html#Syncfusion_UI_Xaml_Calendar_SfCalendarDatePicker_SelectionHighlightMode) property to update the background and border of the dates in a drop-down calendar by highlighting today and a selected date. The default value of `SelectionHighlightMode` property is **Outline**.
```
Comment (mohamedks) [inline winui/Calendar-Date-Picker/Date-Selection-And-Restriction.md:187]: Meaning is changed and not clear now

File: winui/Calendar-Date-Picker/Getting-Started.md [hunk 7]
```diff
@@ -272,9 +272,9 @@ private void SfCalendarDatePicker_SelectedDateChanged(object sender, SelectedDat
...
 
-If you want to perform the validation after the user completely entered their date inputs, set the [EditMode](https://help.syncfusion.com/cr/winui/Syncfusion.UI.Xaml.Calendar.SfCalendarDatePicker.html#Syncfusion_UI_Xaml_Calendar_SfCalendarDatePicker_EditMode) property value as **Normal**. Then the entered date value is validated with the `DisplayDateFormat` property value by pressing the `Enter` key or lost focus. If entered value is not suit with `DisplayDateFormat` property, the previously selected date value sets to `SelectedDate` property.
+If you want to perform the validation after the user completely entered their date inputs, set the [EditMode](https://help.syncfusion.com/cr/winui/Syncfusion.UI.Xaml.Calendar.SfCalendarDatePicker.html#Syncfusion_UI_Xaml_Calendar_SfCalendarDatePicker_EditMode) property value as **Normal**. Then, the entered date value is validated with the `DisplayDateFormat` property value by pressing the `Enter` key or lost focus. If the entered value does not match the `DisplayDateFormat` property, the previously selected date value is set to the `SelectedDate` property.
```
Comment (mohamedks) [inline winui/Calendar-Date-Picker/Getting-Started.md:277]: Match is not a right word here.

File: winui/Calendar-Date-Picker/Getting-Started.md [hunk 8]
```diff
@@ -294,19 +294,19 @@ sfCalendarDatePicker.EditMode = DateTimeEditMode.Normal;
...
+The `SelectedDateChanging` event will be triggered as soon as a date is selected but before the `SelectedDate` property is updated. If the change is considered invalid, it can be canceled. The `SelectedDateChanging` event contains the following properties:
 
-* `OldDate` - Gets a date which is previously selected.
-* `NewDate` - Gets a date which is currently selected.
+* `OldDate` - Gets a date, which is previously selected.
+* `NewDate` - Gets a date, which is currently selected.
 * `Cancel` - Gets or sets whether to cancel the selected date value update.
 
-Users are restricted to select a blackout date from dropdown, however user can give text input through editor. As selecting a blackout date leads to crash, we can cancel the change using `SelectedDateChanging` event.
+Users are restricted to select a blackout date from a drop-down menu, but they can provide text input by the editor. Because selecting a blackout date leads to crash, you can cancel the change using the `SelectedDateChanging` event.
```
Comment (mohamedks) [inline winui/Calendar-Date-Picker/Getting-Started.md:307]: do not use the word they. We should address the customer using "you"

File: winui/Calendar-DateRange-Picker/Restrict-DateRange-Selection.md [hunk 9]
```diff
@@ -194,4 +194,11 @@ sfCalendarDateRangePicker.MinDatesCountInRange = 28;
 
 ![display-mode-in-winui-calendar-date-range-picker](Images/navigate-between-views/display-mode-in-winui-calendar-date-range-picker.png)
 
-N> Download demo from [Github](https://github.com/SyncfusionExamples/syncfusion-winui-tools-calendar-daterange-picker-examples/blob/main/Samples/Selection)
\ No newline at end of file
+N> Download demo from [Github](https://github.com/SyncfusionExamples/syncfusion-winui-tools-calendar-daterange-picker-examples/blob/main/Samples/Selection).
+<<<<<<< .mine
```
Comment (mohamedks) [inline winui/Calendar-DateRange-Picker/Restrict-DateRange-Selection.md:198]: Clear the conflicts. Check if there are more issues like this.

File: winui/Calendar/Localization-And-Formatting.md [hunk 10]
```diff
@@ -181,4 +182,9 @@ sfCalendar.NumberOfWeeksInView = 3;
 
 ![change-number-of-weeks-in-a-view-in-winui-calendar](Images/localization-and-formatting/change-number-of-weeks-in-a-view-in-winui-calendar.png)
 
-N> Download demo application from [GitHub](https://github.com/SyncfusionExamples/syncfusion-winui-tools-calendar-examples/blob/main/Samples/Selection)
\ No newline at end of file
+N> Download demo application from [GitHub](https://github.com/SyncfusionExamples/syncfusion-winui-tools-calendar-examples/blob/main/Samples/Selection).
```
Comment (mohamedks) [inline winui/Calendar/Localization-And-Formatting.md:185]: conflict.

File: winui/Calendar/Overview.md [hunk 11]
```diff
@@ -9,18 +9,18 @@ documentation: ug
...
+The [WinUI Calendar](https://www.syncfusion.com/winui-controls/calendar) control allows users to view and interact with a calendar, allowing them to navigate by month, year, decade, or century. A user can select a single date, multiple dates or a range of dates. It supports different date formats and cells UI customization. Date selection can be restricted by specifying minimum and maximum dates. Specific dates can also be disabled from selection.
 
 ## Control structure
 
 ![winui-calendar-control](Images/getting-started/winui-calendar-control.png)
 
 ## Key Features
 
-* Calendar supports different culture and language types.
+* The calendar supports different cultures and language types.
```
Comment (mohamedks) [inline winui/Calendar/Overview.md:20]: Why do we change casing for some and keep the same case for some?

File: winui/Calendar/Selection.md [hunk 12]
```diff
@@ -76,15 +76,15 @@ public class ViewModel
 {% endhighlight %}
 {% endtabs %}
 
-N> If `SelectedDate` property is used to select a date when `SelectionMode` property value is **Single**. The`SelectedDates` collection property will have only `SelectedDate` property value. 
+N> If the `SelectedDate` property is used to select a date and `SelectionMode` property value is **Single**. The`SelectedDates` collection property will have only `SelectedDate` property value.
```
Comment (mohamedks) [inline winui/Calendar/Selection.md:79]: Should be merged as a single sentence.

File: winui/Calendar/Selection.md [hunk 13]
```diff
@@ -101,17 +101,17 @@ sfCalendar.SelectionMode = CalendarSelectionMode.Multiple;
 {% endhighlight %}
 {% endtabs %}
 
-N> If `SelectedDate` property is used to select a date when `SelectionMode` property value is **Multiple**. The`SelectedDates` collection property will have only `SelectedDate` property value. 
+N> If `SelectedDate` property is used to select a date and `SelectionMode` property value is **Multiple**. The`SelectedDates` collection property will have only `SelectedDate` property value.
```
Comment (mohamedks) [inline winui/Calendar/Selection.md:104]: Merge this sentence.

File: winui/Calendar/Selection.md [hunk 14]
```diff
@@ -130,11 +130,11 @@ sfCalendar.SelectionMode = CalendarSelectionMode.Range;
...
 
-N> Download demo application from [Github](https://github.com/SyncfusionExamples/syncfusion-winui-tools-calendar-examples/tree/main/Samples/Selection)
+N> Download demo application from [Github](https://github.com/SyncfusionExamples/syncfusion-winui-tools-calendar-examples/tree/main/Samples/Selection).
 
-## Highlight today and selected dates
+## Highlight today and selected date
 
-You can highlight the today and selected date using [SelectionHighlightMode](https://help.syncfusion.com/cr/winui/Syncfusion.UI.Xaml.Calendar.SfCalendar.html#Syncfusion_UI_Xaml_Calendar_SfCalendar_SelectionHighlightMode) property to update the background and border of the dates. The default value of `SelectionHighlightMode` property is **Outline**.
+You can update the background and border of the dates by highlighting the today and selected date with the [SelectionHighlightMode](https://help.syncfusion.com/cr/winui/Syncfusion.UI.Xaml.Calendar.SfCalendar.html#Syncfusion_UI_Xaml_Calendar_SfCalendar_SelectionHighlightMode) property. The default value of `SelectionHighlightMode` property is **Outline**.
```
Comment (mohamedks) [inline winui/Calendar/Selection.md:137]: The meaning is not clear.

File: winui/Calendar/Selection.md [hunk 15]
```diff
@@ -178,4 +178,10 @@ sfCalendar.SelectionShape = SelectionShape.Rectangle;
 
 ![change-shape-of-today-and-selected-date-in-winui-calendar](Images/selection/change-shape-of-today-and-selected-date-in-winui-calendar.png)
 
-N> Download demo application from [GitHub](https://github.com/SyncfusionExamples/syncfusion-winui-tools-calendar-examples/blob/main/Samples/Selection)
\ No newline at end of file
+<<<<<<< .mine
```
Comment (mohamedks) [inline winui/Calendar/Selection.md:181]: Conflicts.

File: winui/Chart/Interactive-Features/Tooltip.md [hunk 16]
```diff
@@ -13,9 +13,9 @@ The Tooltip feature allows you to display any information over a [`ChartSeries`]
 
 ## Define Tooltip
 
-By default, a small box containing the data points y values are displayed as the Tooltip. The y values vary depending on the [`ChartSeries`](https://help.syncfusion.com/cr/WinUI/Syncfusion.UI.Xaml.Charts.ChartSeries.html#). For example, a single y value is usually displayed in [`Column`](https://help.syncfusion.com/cr/WinUI/Syncfusion.UI.Xaml.Charts.ColumnSeries.html#) and [`BarSeries`](https://help.syncfusion.com/cr/WinUI/Syncfusion.UI.Xaml.Charts.BarSeries.html#). 
+By default, a small box containing the data points' y values is displayed as the tooltip. The y values vary depending on the [`ChartSeries`](https://help.syncfusion.com/cr/WinUI/Syncfusion.UI.Xaml.Charts.ChartSeries.html#). For example, a single y value is usually displayed in [`Column`](https://help.syncfusion.com/cr/WinUI/Syncfusion.UI.Xaml.Charts.ColumnSeries.html#) and [`BarSeries`](https://help.syncfusion.com/cr/WinUI/Syncfusion.UI.Xaml.Charts.BarSeries.html#).
```
Comment (mohamedks) [inline winui/Chart/Interactive-Features/Tooltip.md:16]: points' y values is  - sentence is wrong

File: winui/Chart/Interactive-Features/Trackball.md [hunk 17]
```diff
@@ -457,9 +457,9 @@ chart.Behaviors.Add(behavior);
 ![Alignment support for Trackball series label in WinUI Chart](Interactive-Features_images/Interactive-Features_img14.jpeg)
 
 
-### Label Display Mode
+### Label display mode
 
-When there is a multiple series, by default, the Trackball series label will be displayed only for the nearest point. If you want to display all the y values with respect to the x value then the [`LabelDisplayMode`](https://help.syncfusion.com/cr/WinUI/Syncfusion.UI.Xaml.Charts.ChartTrackballBehavior.html#Syncfusion_UI_Xaml_Charts_ChartTrackballBehavior_LabelDisplayMode) property is set to [`FloatAllPoints`](https://help.syncfusion.com/cr/WinUI/Syncfusion.UI.Xaml.Charts.TrackballLabelDisplayMode.html).
+By default, the Trackball series label will be displayed only for the nearest point When there is a multiple series. If you want to display all the y values with respect to the x value, then the [`LabelDisplayMode`](https://help.syncfusion.com/cr/WinUI/Syncfusion.UI.Xaml.Charts.ChartTrackballBehavior.html#Syncfusion_UI_Xaml_Charts_ChartTrackballBehavior_LabelDisplayMode) property is set to [`FloatAllPoints`](https://help.syncfusion.com/cr/WinUI/Syncfusion.UI.Xaml.Charts.TrackballLabelDisplayMode.html).
```
Comment (mohamedks) [inline winui/Chart/Interactive-Features/Trackball.md:462]: When -> when

File: winui/Chart/Legend.md [hunk 18]
```diff
@@ -438,7 +438,7 @@ The [`series`](https://help.syncfusion.com/winui/chart/seriestypes/series) can b
 ![ToggleSeriesVisibility support for legend in WinUI Chart](Legend_images/winui_chart_legend_toggle_series_visibility.png)
 
 
-We can also view the associated [`series`](https://help.syncfusion.com/winui/chart/seriestypes/series), by clicking on its disabled legend item,
+YOu can also view the associated [`series`](https://help.syncfusion.com/winui/chart/seriestypes/series), by clicking on its disabled legend item.
```
Comment (mohamedks) [inline winui/Chart/Legend.md:441]: Clear the error in casing

File: winui/Chart/SeriesTypes/ColumnandBar.md [hunk 19]
```diff
@@ -44,7 +44,7 @@ chart.Series.Add(series);
 
 ## Bar
 
-Bar series are similar to column series, excepts its orientation. The following code examples shows how to use [`BarSeries`](https://help.syncfusion.com/cr/WinUI/Syncfusion.UI.Xaml.Charts.BarSeries.html#).
+Bar series is similar to column series, except for their orientation. The following code example shows how to use [`BarSeries`](https://help.syncfusion.com/cr/WinUI/Syncfusion.UI.Xaml.Charts.BarSeries.html#).
```
Comment (mohamedks) [inline winui/Chart/SeriesTypes/ColumnandBar.md:47]: except for their orientation  -> should be singluar

**Review by mohamedks — APPROVED** [review]

**Pull request merged**